modules/webapp/webapp.bicep

```diff
@@ -25,6 +25,9 @@ param redcapZipUrl string
 param redcapCommunityUsername string
 @secure()
 param redcapCommunityPassword string
+param scmRepoUrl string
+param scmRepoBranch string = 'main'
+param preRequsitesCommand string = 'apt-get install unzip -y && apt-get install -y python3 python3-pip'
```

Comment:
We should not repeat this default value.

Reply:
fixed